Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM php:7.0-apache
+
+RUN apt-get update \
+ && apt-get install -y git zlib1g-dev \
+ && docker-php-ext-install zip \
+ && a2enmod rewrite \
+ && sed -i 's!/var/www/html!/var/www/public!g' /etc/apache2/apache2.conf \
+ && mv /var/www/html /var/www/public \
+ && curl -sS https://getcomposer.org/installer \
+  | php -- --install-dir=/usr/local/bin --filename=composer
+
+WORKDIR /var/www
+
+VOLUME /var/www
```

Comment:
When I started, I was providing both a vanilla `Dockerfile` *and* docker-compose configuration separately. I'll remove it now, as we've decided to recommend docker-compose specifically.